docs/source/getting-started/tabular-data/grouping.ipynb

```diff
@@ -161,7 +161,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Protected Group Keys\n",
+    "## Stable Group Keys\n",
```

Comment:
The paragraph which follows doesn't mention anything about stability, so I'm not really sure why that's the term we're using.